### ISSUE TITLE
Screen Reader Can't Identify Extension Button

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#8269]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
When navigating around Brave with a screen reader, extension buttons aren't distinguishable from other extension buttons. In fact, they aren't even identified as extension buttons to begin with--just interactive elements.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Desktop

- Brave Version (revision SHA):
https://github.com/brave/browser-laptop/commit/f07c1421b982ef66e4cd369e8d928a3f9b19ffd5

- Steps to reproduce (Requires NVDA):
    1. Enable LastPass
    2. Focus address bar
    3. Press tab until LastPass button is identified
    
- Actual result:
LastPass button is never identified.

- Expected result:
User should know when they have an extension button in focus.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

**Comment by cezaraugusto** [comment]
Doesn't work on Ubuntu, which comes with [Orca](https://wiki.gnome.org/action/show/Projects/Orca?action=show&redirect=Orca) built-in. Not sure if I followed the steps correctly so will left closed for now. @jonathansampson mind confirming that? Running Ubuntu 16.04

**Comment by alexwykoff** [comment]
@cezaraugusto the primary concern in this case is Brave vs Chrome.  Voiceover on OS X isn't all that great either.  Longer term, we certainly should push to be a leader in A11Y and if we can do that on Linux then all the better. 😸

**Comment by jonathansampson** [comment]
@alexwykoff @cezaraugusto Testing across Windows, Linux, and macOS is going to be really tough. Though I'm no expert, I recall there being differences among even the various screen readers on any given OS. If we can get 1 reader performing better on 1 OS, that's a win.

In my opinion, we should be prepared to make many iterative changes in this space.

**Comment by cezaraugusto** [comment]
thanks both for clarifying, glad we are heading this way ++

**Comment by alexwykoff** [comment]
Works great for Jaws. Thanks @jonathansampson !
